docs/development.md

```diff
@@ -34,49 +34,109 @@ To summarize the steps below, everything you need to do:
 5. Later on: Ensure that your homepages link to `/Settings/Options/Settings/GlobalLink`
     in their linked settings.
 
-## Adding GCC Submodule
+### Branches
 
-The GCC extension can be added the Blueprint workspace as a Git submodule as follwos:
-```bash
-$ mkdir -p modules/extensions
-$ cd modules/extensions
-$ git submodule add https://github.com/CoreMedia/coremedia-globallink-connect-integration.git gcc
-$ git submodule init
-$ git checkout -b <tag-name> <your-branch-name>
+![Branch Model](img/branch-model.png)
+
+* **main:** Will be initially used to create `develop` branch. Afterwards,
+  it will just be used to merge changes from `develop` branch to `main`,
+  i.e., it will just be recipient afterwards. On _release_ the main merge
+  commit will be tagged. See below for details on tagging.
+
+* **develop:** After initial creation, all development by CoreMedia and
+  merging pull request will happen here.
+
+* **ci/develop:** An artificial branch required for CoreMedia CI systems. It is
+  required, as for CoreMedia CI we need to change the parent POMs in that way,
+  that we set the version to `9999.9` and add a relative path, so that
+  it matches our workspace setup.
+
+  As soon as changes from `develop` shall be published to CI, we rebase
+  the adaptions:
+
+    ```bash
+    $ git checkout "ci/develop"
+    $ git rebase "origin/develop"
+    $ git push --force-with-lease
+    ```
```

Comment:
Code-block should be in line with previous paragraph:

```suggestion
  As soon as changes from `develop` shall be published to CI, we rebase
  the adaptions:

  ```bash
  $ git checkout "ci/develop"
  $ git rebase "origin/develop"
  $ git push --force-with-lease
  ```
```